### DISCUSSION TITLE
Using Zoom together with SEB?

### DISCUSSION THREAD

**Discussion by Timo1993** [SafeExamBrowser/seb-mac#77] [Q&A] [answered]
Hello, we think about using the SafeExamBrowser for an exam. Our idea is to control the identity of the students via Zoom. We already tested it on a Windows computer which worked well. Is it also possible to use Zoom with the SafeExamBrowser (as Zoom might be killed and there are no permitted processes on MacOS)? If not: Is there a workaround to use Zoom and SEB on a Mac computer?

Kind regards

**Comment by danschlet** [comment, marked as answer]
Hi, this question was discussed in our other forum on SourceForge: https://sourceforge.net/p/seb/discussion/844844/thread/2e8caa51f3/?limit=25&page=1#b51a

You can use Zoom with SEB for macOS, but you have to change settings accordingly (see the posting above), it has to be started and connected manually before starting SEB and students cannot interact with the Zoom app during the exam.

**Reply by danschlet** [reply]
PS: Screen sharing in the Zoom client currently cannot be used together with SEB for macOS. We will add some settings for permitted processes which would probably allow this in a future version. Also SEB 3.2 for macOS will have an integration with Zoom, so SEB will be able to directly join Zoom meetings configured in its settings or controlled by SEB Server (similar to the existing integration with Jitsi Meet in SEB 3.0 for iOS and SEB 3.2 for Windows).
